### ISSUE TITLE
About the EPSG projection information

### ISSUE THREAD

**Issue by basbaba** [nutonomy/nuscenes-devkit#966]
In  nuscenes-devkit/python-sdk/nuscenes/map_expansion/map_api.py
I see:
"We use the same WGS 84 Web Mercator (EPSG:3857) projection as Google Maps/Earth."

So I converted the 4 maps(1 in Boston and 3 in Singapore) from EPSG:3857 to EPSG:4326, and export them to the GeoServer(A GIS system), the result is: the Singapore's nuScenes maps are matched with OpenStreetMap(OSM) perfectly, but the Boston is not.

In the SDK, there are 4 cities reference coordination:
    "boston-seaport": [42.336849169438615, -71.05785369873047],
    "singapore-onenorth": [1.2882100868743724, 103.78475189208984],
    "singapore-hollandvillage": [1.2993652317780957, 103.78217697143555],
    "singapore-queenstown": [1.2782562240223188, 103.76741409301758],
First, I think maybe it's a problem of boston-seaport's reference coordination, so I tried to give it an offset, but the result is still unmatched with OSM. At last I found  Boston nuScenes map is not only having Offset problem but also Scaled problem.
So I'm in doubt: the Boston nuScenes map is projected under EPSG:3857. If it's not, what EPSG is with Boston nuScenes map?

The fist picture is composed of nuScenes Boston map and OSM and Overture map:
![boston_nuscenes_osm](https://github.com/nutonomy/nuscenes-devkit/assets/8930126/f542856c-99f0-4c2a-bc45-2c98240b9eae)

Red lines are roads from OSM, Grey blocks are buildings from Overture, Green polygons are from nuScenes, they both are projected into EPSG:4326, you can find out Green polygons are unmatched with Red lines.
As a reference, this is the same area from google map:
![boston_google](https://github.com/nutonomy/nuscenes-devkit/assets/8930126/21e65500-6e85-4a49-ba9f-539b78a26d54)

Let me set several reference points on the first image to show where nuScenes Boston Green polygons should be at in correct position:
![boston_nuscenes_osm_correct](https://github.com/nutonomy/nuscenes-devkit/assets/8930126/aec93116-02d8-4223-98bd-6ae38e4f4d4f)

For a temporary solution, I did the following:
1. scale the nuScenes Boston map polygons' coordinates with factor=1.35 as scaled_polygon
2. convert "boston-seaport": [42.336849169438615, -71.05785369873047] to EPSG:3857 as ref_coords_3857
3. scaled_polygon_on_map = scaled_polygon + ref_coords_3857
4. converted scaled_polygon_on_map from EPSG:3857 to EPSG:4326
5. import the results into GeoServer and get this result:
![boston_nuscenes_osm_scaled](https://github.com/nutonomy/nuscenes-devkit/assets/8930126/a43e984b-9eb9-4e07-a548-d67d2fe5acba)
It looks better but not a thorough solution. I need to know: is nuScenes Boston map is under EPSG:3857 or something else or I did some mistakes?
The last image is nuScenes Singapore map without polygon coordinates scaled(Only converted from EPSG:3857 to EPSG:4326):
![singapore_nuscenes_osm](https://github.com/nutonomy/nuscenes-devkit/assets/8930126/ab4763f1-8afc-4406-806b-ebdfad105568)

Thanks!

**Comment by basbaba** [comment]
I did another test:
1. keep nuScenes Boston map unmodified
2. transform Overture buildings from EPSG:4326 to EPSG:3857 as building_3857
3. scale down building_3857 with factor=1./1.35
4. overlay the scaled building_3857 on nuScenes Boston map
I got a right maps:
![boston_nuscenes_osm_3857_scaled](https://github.com/nutonomy/nuscenes-devkit/assets/8930126/9ad6b12a-a032-444a-8ca2-91206eba3d5a)

So are you sure the Boston map is projected under EPSG:3857?

Best Regards!
